plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/FormalVerificationPluginErrorMessages.kt

```diff
@@ -37,5 +37,10 @@ object FormalVerificationPluginErrorMessages : BaseDiagnosticRendererFactory() {
             "Function may return a {0} value.",
             CommonRenderers.STRING
         )
+        put(
+            PluginErrors.INVALID_INVOCATION_TYPE,
+            "Function may not be called {0} time(s).",
```

Comment:
This makes it sound like calling it that many times is wrong; a formulation like "Could not verify that ..." would be clearer.